Bugzilla/Mailer.pm

```diff
@@ -182,6 +185,21 @@ sub MessageToMTA {
     Bugzilla::Hook::process('mailer_before_send',
                             { email => $email, mailer_args => \@args });
 
+    try {
+        my $to         = $email->header('to') or die 'Unable to find "To:" address';
+        my @recipients = Email::Address->parse($to);
+        die qq{Unable to parse "To:" address - $to} unless @recipients;
+        foreach my $to_address (@recipients) {
+            my $badhosts   = Bugzilla::Bloomfilter->lookup("badhosts") or die "No badhosts bloomfilter";
```

Comment:
Is this something that must be done in the inner loop? Or can we move this outside?

Reply:
It's unlikely to ever be a loop of more than one iteration.
Perhaps it would be more clear to make it an error if `@recipients > 1`?

Reply:
@dylanwh So this code is run several times with each person to be emailed, I see. I thought for a minute that `@recipients` variable would have a list of all people receiving the email, but, yeah if it's like a normal email To field then there'll likely be just one address. I'll approve this and leave it to you if you want to clarify that.